src/Serilog.Sinks.OpenTelemetry/Sinks/OpenTelemetry/HttpExporter.cs

```diff
@@ -36,9 +36,12 @@ sealed class HttpExporter : IExporter, IDisposable
     /// <param name="headers">
     /// A dictionary containing the request headers.
     /// </param>
-    public HttpExporter(string endpoint, IDictionary<string, string>? headers)
+    /// <param name="httpMessageHandler">
+    /// Custom HTTP message handler.
+    /// </param>
+    public HttpExporter(string endpoint, IDictionary<string, string>? headers, HttpMessageHandler? httpMessageHandler = null)
     {
-        _client = new HttpClient();
+        _client = new HttpClient(httpMessageHandler ?? new HttpClientHandler());
```

Comment:
The default HTTP client handler differs in a few scenarios; this will be safer as:

```csharp
_client = httpMessageHandler == null ? new HttpClient() : new HttpClient(httpMessageHandler);
```

(E.g. see https://learn.microsoft.com/en-us/dotnet/api/system.net.http.socketshttphandler?view=net-8.0#remarks)